.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,88 @@
+name: Integration tests
+on:
+  push:
+    branches: [ main, master, jsign-integration-test ]
+    schedule:
+      - cron: '0 0 * * *'
+env:
+  CARGO_TERM_COLOR: always
+
+jobs: 
+  witness-generator:
+    name: Generate EEST benchmark fixtures
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@nightly
+        with:
+          components: clippy, rustfmt
+      
+      - name: Download and Extract Fixtures
+        run: |
+          chmod +x ./scripts/download-and-extract-fixtures.sh
+          ./scripts/download-and-extract-fixtures.sh
+      
+      # This step is temporary until a new benchmark release is done.
+      - name: Delete prague fixtures
+        run: rm -rf ./zkevm-fixtures/fixtures/blockchain_tests/prague
+
+      - name: Generate benchmark fixtures
+        run: RUST_LOG=info cargo run -p witness-generator-cli --release -- tests --include keccak --include Prague
```

Comment:
For all runs I use `--release` since debug is very slow for the runs.
The downside is that compilation takes longer, but I think that is still better than faster debug compilation but slower run.